Wordle.cpp

```diff
@@ -0,0 +1,256 @@
+#define _CRT_SECURE_NO_WARNINGS
+
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <cstdlib>
+#include <ctime>
+
+enum class GameMode {
+    exit = 0,
+    wordleOfTheDay = 1,
+    randomWordle = 2
+};
+
+constexpr int WORD_SIZE = 5;
+constexpr int WORD_ARRAY_SIZE = 6;
+constexpr int WORDS_COUNT = 3374;
```

Comment:
it can be a part of file. If the file for some reason will have different number of lines the program will crash